bay-services/gremlin.yaml

```diff
@@ -15,3 +16,21 @@ services:
       REPLICAS: 1
   path: /openshift/template.yaml
   url: https://github.com/fabric8-analytics/gremlin-docker/
+
+- <<: gremlin_def
```

Comment:
Ah, I think the correct syntax should be:
```
- <<: *gremlin_def
```
I totally missed the asterisk the first time.

Reply:
Yeah, just noticed that. Fixed.